app.py

```diff
@@ -33,20 +56,18 @@ def create_app(testing=False) -> Flask:
     flask_app.jinja_env.trim_blocks = True
     flask_app.jinja_env.lstrip_blocks = True
 
-    if testing:
-        flask_app.config.from_object(
-            "config.environments.development.DevelopmentConfig"
+    if (os.environ.get("FLASK_ENV") == "development") | testing:
```

Comment:
just a Q - where do we set the "FLASK_ENV" env var?

Reply:
@Adam-W1 I think either in `.flaskenv` file or overridden in an environment variable